src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -60,4 +56,65 @@ alias_netmaskbit6_tooltip : T('Select the prefix length used on the network.'),
 delete_placeholder6: T('Delete'),
 delete_tooltip6: T('Set to delete this alias.'),
 
+bridge_members_placeholder: T('Bridge Members'),
+bridge_members_tooltip: T(''),
+
+failover_critical_placeholder: T('Failover Critical'),
+failover_critical_tooltip: T(''),
+
+failover_group_placeholder: T('Failover Group'),
+failover_group_tooltip: T(''),
+
+failover_vhid_placeholder: T('Failover VHID'),
+failover_vhid_tooltip: T(''),
+
+failover_alias_address_placeholder: T('Failover Alias'),
+failover_alias_address_tooltip: T(''),
+
+failover_virtual_alias_address_placeholder: T('Failover Virtual Alias'),
+failover_virtual_alias_address_tooltip: T(''),
```

Comment:
From TrueNAS Guide "Failover" section:

"IP Address used for administrative access to this array."

Reply:
We should probably call IPs in the UI.

This is the same as Virtual IP in legacy UI, its the CARP address (shared between nodes)

Reply:
how about we call it Failover Virtual IP Address @william-gr @wblock  (and perhaps just change the word Alias to IP Address on all fields that say Alias?)

Reply:
The tooltip sounds fine to me though, what do you say @wblock

Reply:
yes

Reply:
The intent if this Ip is not just to administrative access it, its the IP shared between the controllers.